### PR TITLE
Release akka-http 10.0.3

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: news/_posts/2017-01-26-akka-http-10.0.3-released.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+we — the Akka HTTP committers — are happy to announce the availability of the third maintenance release of Akka HTTP 10.0.
+This release also includes the security fix that was released earlier this week as Akka HTTP 10.0.2.
+
+The most notable changes in this release are:
+
+ * The addition of the experimental HttpApp feature to write simple Akka HTTP server with as little boilerplate as possible.
+   Thanks a lot, [@jlprat](https://github.com/jlprat) for contributing this feature and carrying through the PR with the
+   most comments so far!
+ * Several bug fixes.
+ * Lots of smaller and larger documentation fixes contributed by the community.
```
Comment (ktoso) [inline news/_posts/2017-01-26-akka-http-10.0.3-released.md]: + continued work on the HTTP/2 server internals, not ready for public consumption yet but can be seen as developer preview?

File: news/_posts/2017-01-26-akka-http-10.0.3-released.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Please see the full change log at the [release notes in the documentation](http://doc.akka.io/docs/akka-http/10.0.3/release-notes.html).
```
Comment (ktoso) [inline news/_posts/2017-01-26-akka-http-10.0.3-released.md]: cool

File: news/_posts/2017-01-26-akka-http-10.0.3-released.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+We also made a lot of progress on the upcoming experimental Http/2 support for Akka HTTP. Stay tuned!
```
Comment (ktoso) [inline news/_posts/2017-01-26-akka-http-10.0.3-released.md]: add it in the `*` section, people will miss it otherwise i think

Comment (jrudolph) [inline news/_posts/2017-01-26-akka-http-10.0.3-released.md]: Changed especially for the people with the two-line attention span ;)

**Comment by ktoso** [comment]
LGTM

File: news/_posts/2017-01-26-akka-http-10.0.3-released.md (hunk 3 above)
Comment (ktoso) [inline news/_posts/2017-01-26-akka-http-10.0.3-released.md]: That's me ;-)

**Pull request merged**